apps/desktop/src/i18n/messages.ts

```
@@ -914,6 +956,32 @@ const enUSMessages: Record<keyof typeof zhCNMessages, string> = {
+    'settings.memory.saved': 'Memory saved',
+    'settings.memory.saveFailed': 'Failed to save memory',
+    'settings.memory.deleteFailed': 'Failed to delete memory',
+    'settings.memory.toggleFailed': 'Failed to update memory enabled state',
+    'settings.memory.toggleTitle': 'Enable or disable',
+    'settings.memory.resizeList': 'Resize memory list width',
+    'settings.memory.actions.create': 'New memory',
+    'settings.memory.empty.title': 'No memories yet',
+    'settings.memory.updatedAt': 'Last updated: {time}',
+    'settings.memory.createDefaultTitle': 'New memory',
+    'settings.memory.createDefaultApplicability':
+        'Describe when this memory should be used here.',
```

ESLint

[error] Delete `⏎·······`

(prettier/prettier)